### ISSUE TITLE
addMultipleMediaFromRequest don't recognize array of same name

### ISSUE THREAD

**Issue by Epistol** [spatie/laravel-medialibrary#1450]
Hello there, after multiple tries, looks like the library doesn't take the case where multiple files from the same request under an array.

Example from library : 

```php
$fileAdders = $this->model
    ->addMultipleMediaFromRequest(['file-one', 'file-two'])
    ->each(function ($fileAdder) {
        $fileAdder->toMediaCollection();
    });
```

What I would like to do : 

```php
$fileAdders = $this->model
    ->addMultipleMediaFromRequest(['file[]']) // or (, 'file[0]', 'file[1]', etc...)
    ->each(function ($fileAdder) {
        $fileAdder->toMediaCollection();
    });
```

I know it looks like a specific use case, but I don't know if it's just me that didn't find anything in the source of the lib, or that there is nothing possible for that for now.

Cordially,
Epistol

**Comment by brendt** [comment]
Hi Epistol

Sounds like a good feature we could add. Though I'm unsure about the `file[]` notation. To me personally it would make more sense to use Laravel's naming conventions and use `file.*`. That's an implementation detail though.

Do you think you'd want to give it a try and submit a PR for this? You'd need to make changes in `\Spatie\MediaLibrary\FileAdder\FileAdderFactory::createMultipleFromRequest`. No pressure though, let me know if you're not up for it and I'll be happy to make a PR myself.

**Comment by codingnninja** [comment]
/claim ... I want to do this.

**Comment by brendt** [comment]
Go for it!

**Comment by ahmedofali** [comment]
@brendt please review the pull request

**Comment by brendt** [comment]
For reference: #1473

**Comment by brendt** [comment]
Can you provide a link to the PR?

**Comment by Epistol** [comment]
Looks like https://github.com/spatie/laravel-medialibrary/pull/1471 but travis ci doesn't seem happy with it :')

**Comment by ahmedofali** [comment]
@brendt Yes because of this issue [1472](https://github.com/spatie/laravel-medialibrary/issues/1472) and I submitted it to mention that there is a problem with links and @riasvdv mentioned that it's fixed

**Comment by brendt** [comment]
Fixed in 7.6.2: https://github.com/spatie/laravel-medialibrary/releases/tag/7.6.2
